sympy/core/power.py

```diff
@@ -1735,12 +1735,10 @@ def mul(d1, d2):
                 raise NotImplementedError()
         if not d.is_positive:
             g = g.simplify()
-            _, d = g.leadterm(x)
-            if not d.is_positive:
-                if g.is_zero:
+            if g.is_zero:
                     return f**e
-                g = ((b - f)/f).expand()
-                _, d = g.leadterm(x)
```

Comment:
The indentation of these lines (after `if not d.is_positive:`) should be preserved.

Reply:
Do you mean something like this?
```diff
diff --git a/sympy/core/power.py b/sympy/core/power.py
index cefba7fe5c..f179650b97 100644
--- a/sympy/core/power.py
+++ b/sympy/core/power.py
@@ -1736,7 +1736,7 @@ def mul(d1, d2):
         if not d.is_positive:
             g = g.simplify()
             if g.is_zero:
-                    return f**e
+                return f**e
             g = ((b - f)/f).expand()
             _, d = g.leadterm(x)
             if not d.is_positive:
```